data/index

```diff
@@ -1,7 +1 @@
-{
-  "1": [2, 3],
-  "2": [],
-  "3": [4, 5],
-  "4": [],
-  "5": []
-}
+{}
```

Comment:
Dont commit. Use git add -p next time to selectively stage only the changes you want to include